README.md

```diff
@@ -53,12 +53,8 @@ switch to the browser code editor.
   parseIgnore                 list assets matched by pcignore.txt
 ```
 
-To automatically answer "yes" to any prompts that might print
-on the command line from `pushAll` or `pullAll` commands, simply supply `-y` or `--yes` arguments:
-
-```
-pcsync pushAll -y
-```
```

Comment:
Any reason to remove the CLI example?

Reply:
I felt it made that paragraph longer than necessary